config/os.go

```diff
@@ -44,7 +44,6 @@ func GetEOL(family, release string) (eol EOL, found bool) {
 			"2":    {StandardSupportUntil: time.Date(2025, 6, 30, 23, 59, 59, 0, time.UTC)},
 			"2022": {StandardSupportUntil: time.Date(2026, 6, 30, 23, 59, 59, 0, time.UTC)},
```

Comment:
Can we remove this?

Reply:
Amazon Linux 2022 is only available for a moment, but since it has been released, this part cannot be erased.